pennylane/transforms/metric_tensor.py

```diff
@@ -864,14 +871,18 @@ def _get_aux_wire(aux_wire, tape, device_wires):
         and an often reasonable choice else.
     """
     if aux_wire is not None:
+        aux_wire = qml.wires.Wires(aux_wire)[0]

```

Comment:
What does this line do, actually? Is it because we want a single wire label, and the input might be a wires object or a wire label? I think the docstring could be clearer here about what input type `aux_wire` can be.